ex5.1_this_is_the_way.py

```diff
@@ -0,0 +1,12 @@
+import os
+INPUT_FOLDER_MESSAGE="insert the path to the folder:"
+# function that print the file in folder that start with "deep"
+def print_deep_folder():
```

Comment:
function documentation should look like this -
![Capture](https://user-images.githubusercontent.com/13467749/159840350-74d1ada2-17b6-43c2-94e8-9fbb13308117.JPG)